src/main/java/org/beanpod/switchboard/dao/StreamDaoImpl.java

```diff
@@ -52,6 +70,17 @@ public StreamEntity updateStream(StreamDto streamDto) {
     return streamRepository.save(streamEntity);
   }
 
+  public StreamStatDto updateStreamStat(StreamStatDto streamStatDto) {
+    if (!streamRepository.existsById(streamStatDto.getId())) {
+      throw new StreamDoesNotExistException(streamStatDto.getId());
+    }
+    Optional<StreamStatDto> streamStat = getStreamStat(streamStatDto.getId());
+    statMapper.updateStreamStatFromDto(streamStatDto, streamStat.orElse(null));
```

Comment:
Can you make it accept partial updates like we did for update device? We might end up getting stats from the sender and the receiver, so each will be pushing only some of the values.

Reply:
Yes it does  accept partial updates

Reply:
You were right it didnt acccept partial values although i thought it was,
thats because i used long instead of Long. Now it works!